conans/client/remote_manager.py

```diff
@@ -279,7 +279,10 @@ def unzip_and_get_files(files, destination_dir, tgz_name, output):
     check_compressed_files(tgz_name, files)
     if tgz_file:
         uncompress_file(tgz_file, destination_dir, output=output)
-        os.remove(tgz_file)
+        # move the .tgz to another place
+        tgzdir = destination_dir + "_tgz"
+        mkdir(tgzdir)
+        shutil.move(tgz_file, os.path.join(tgzdir, tgz_name))
```

Comment:
The problem I see storing the file in another folder is some cases like this that you are using revisions and do this operations:

```
conan new hello/0.1
conan create .
conan upload hello/0.1 -r artifactory -c --all
conan download hello/0.1@ -r artifactory 
echo "# new revision" >> conanfile.py 
conan export .
```

You end up with a folder `hello/0.1/_/_/package/package_id_tgz`, but the original has been removed because Conan already took care of removing that folder in certain situations... so it's necessary now to also take care of handling this folder which maybe adds a little bit more complexity.

Before Conan was removing the package_id folder so the downloaded file would also be deleted.